package.json

```diff
@@ -4,10 +4,20 @@
   "description": "",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/index.js"
+    },
+    "./dist/gettext": {
+      "import": "./dist/esm/gettext.js",
+      "require": "./dist/gettext.js"
+    }
+  },
   "scripts": {
-    "build": "babel ./lib --out-dir dist --extensions '.ts,.tsx' --source-maps && tsc --emitDeclarationOnly",
```

Comment:
does this mean we no longer export type declarations?

Reply:
No we emit them **together** with the ESM files.
So instead of **only** emit declarations we emit ESM js and declarations together :)